ui/sign-in.reel/sign-in.html

```diff
@@ -71,13 +71,25 @@
                 "element": {"#": "submit"},
                 "value": "Sign In"
             }
+        },
+
+        "browserMessage": {
+            "prototype": "montage/ui/text.reel",
+            "properties": {
+                "element": {"#": "browserMessage"},
+                "value": "We're sorry but at this time FreeNAS only supports the Chrome browser."
+            },
+            "bindings": {
+                "classList.has('hide')": {"<-": "@owner.isBrowserSupported"}
```

Comment:
I am using hide/show. `is-hidden` looks and sounds like a boolean in which I don't know the answer to. `hide` is a clear statement.

Let me know if you disagree.r